rs/nns/governance/src/lib.rs

```diff
@@ -243,13 +244,24 @@ impl IcClock {
     }
 }
 
+fn time_u64_nanos() -> u64 {
```

Comment:
Since we almost everywhere use timestamp_seconds, it is probably better to do something like

```
fn now_timestamp_seconds() -> u64
```

# Tangent

In an ideal world,

```
fn now() -> SystemTime { ...
```

... because raw numbers bad

... but this is less convenient for us, since we use *_timestamp_seconds everywhere.

Reply:
changed it to be more consistent with other usages.